plugin/eleline.vim

```diff
@@ -222,7 +222,13 @@ function! ElelineCoc() abort
     return ''
   endif
   if get(g:, 'coc_enabled', 0)
-    return coc#status().' '
+    let l:coc_status = coc#status()
+    if !empty(get(b:, 'coc_git_blame', ''))
+      if len(l:coc_status) + len(b:coc_git_blame) < winwidth("%") / 3
+        return l:coc_status.' '.b:coc_git_blame
```

Comment:
I think the full blame info can be very long and consume too many valuable spaces of the statusline. If you only show them when the info's short, that's kind of strange as they can just show occasionally.

Reply:
yes @liuchengxu  currently it only shows when the info isn't too long. I'm actually not sure on the correct way to see git blame.

in virtual text it can be too distracting while coding
in status line it looks better but maybe not enough space

Reply:
You can truncate the blame info to a short one, but not sure that's still very useful.